tutorials/nservicebus-step-by-step/1-getting-started/tutorial.md

```diff
@@ -20,7 +20,7 @@ In this first lesson, which should take 10-15 minutes, you will create your firs
 
 Let's build something simple to give NServiceBus a try.
 
-This tutorial uses NServiceBus version 8, .NET 6, and assumes an up-to-date installation of [Visual Studio 2022](https://www.visualstudio.com/downloads/).
+This tutorial uses NServiceBus version 9, .NET 8, and assumes an up-to-date installation of [Visual Studio 2022](https://www.visualstudio.com/downloads/).
 
 > [!NOTE]
```

Comment:
Not sire of this note make sense since since this is targeted at user learning to use our latest version?